astro/src/content/quickstarts/quickstart-javascript-nextjs-web.mdx

```diff
@@ -0,0 +1,232 @@
+* Your admin username is `admin@example.com` and the password is `password`.
+* The base URL of FusionAuth `http://localhost:9011/`.
+
+You can log into the [FusionAuth admin UI](http://localhost:9011/admin) and look around if you want, but with Docker/Kickstart you don’t need to.
+
+### Create Next.js Application
+
+In this section, you'll set up a basic Next.js application with two pages. 
+
+1. Homepage
+1. Account - protected
+
+Create a new application using the `npx`.
+
+```shell
+npx create-next-app@latest changebank --ts --eslint --no-tailwind --src-dir --app --import-alias "@/*"
+```
+
+Make sure you are in your new directory `changebank`.
+
+```shell
+cd changebank
+```
+
+Install [NextAuth.js](https://next-auth.js.org/getting-started/example), which simplifies integrating with FusionAuth and creating a secure web application.
+
+```shell
+npm install next-auth
+```
+
+Copy environment variables from our complete application example.
+
+```shell
+cp ../complete-application/.env.example .env.local
+```
+
+Also copy an image file into a new directory within `public` called `img`.
+
+```shell
+mkdir ./public/img && cp ../complete-application/public/img/money.jpg ./public/img/money.jpg
+```
+
+As you will be recreating all of the files in our app directory, please delete all files within `/src/app`.
+
+```shell
+rm -rf ./src/app && mkdir ./src/app
+```
+
+## Authentication
+
+Next.js 13.2 introduced <a href="https://nextjs.org/docs/app/building-your-application/routing/router-handlers" target="_blank">Route Handlers</a>, which are the preferred way to handle REST-like requests. In the `Changebank` application you can configure NextAuth.js FusionAuth's provider in a new route handler by creating a file within `src/app/api/auth/[...nextauth]/route.ts`.
+
+On first load of Next.js this file will make sure that you have all of the correct environment variables. The variables are then exported in an object called `authOptions` which can be imported on the server when you need to get our session using `getServerSession`.
+
+The `FusionAuthProvider` is then provided to `NextAuth` as a provider for any `GET` or `POST` commands that are sent to the `/api/auth/*` route.
+
+Create a new file named `src/app/api/auth/[...nextauth]/route.ts` and copy the following CSS for the ChangeBank application.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/src/app/api/auth/[...nextauth]/route.ts"}
+  lang="ts" />
+
+## App Customization
+
+### Styles
+
+Create a new file named `src/app/globals.css` copy the below CSS for the ChangeBank application.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/src/app/globals.css"}
+  lang="ts" />
+
+### Login Button
+
+Create a new file in `src/components/login-button.tsx` that will be used for a button component. Our login button will only be used on the client side so make sure to add `use client` at the top of this file.
```

Comment:
I think it needs to be `LoginButton.tsx`. That is what it is in `complete-application` and I get an error when working through these steps.